README.md

```diff
@@ -45,8 +60,10 @@ var Panda = function(name, age) {
 }
 ```
 Your Answer:
-```js
-// your code here
+```
+Panda.prototype = function() {
```

Comment:
So close! We would just want to make sure to define the method on the prototype like this: 
```
Panda.prototype.eat_bamboo = function(){
  this.num_bamboo_eaten += 1;
}
```